src/main/java/com/gengo/client/JsonHttpApi.java

```diff
@@ -169,6 +170,7 @@ protected JSONObject call(String url, HttpMethod method, JSONObject data, boolea
                 {
                     error = error.getJSONArray("0").getJSONObject(0);
                 }
+                System.out.println(error.toString());
```

Comment:
Remove it before release.

But you can work on it in another PR as we discussed above.